src/muya/themes/codemirror/one-dark.css

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
I think you should put all theme files in `renderer` folder, not the `muya` folder. Because all the themes all belongs to Mark Text, not muya. and only leave a default css file to `muya`.

Reply:
You mean all CodeMirror themes, right?

Reply:
I mean all the Mark Text themes, because the theme file contains style which only for renderer process, 

```css
  /*marktext*/
  --sideBarColor: rgba(0, 0, 0, .6);
  --sideBarTitleColor: rgba(0, 0, 0, 1);
  --sideBarTextColor: rgba(0, 0, 0, .4);
  --sideBarBgColor: rgba(242, 242, 242, 0.9);
  --sideBarItemHoverBgColor: rgba(0, 0, 0, .03);
  --itemBgColor: rgba(255, 255, 255, 0.6);
```

for example, the code above maybe not used in muya.

Reply:
I did so. The basic theme and light prismjs theme is still in muya and the remaining themes are in the renderer folder.

Reply:
I still see all the themes in the muya folder, did you push your codes?